agent/custom/action/warehouse_inventory.py

```
@@ -0,0 +1,231 @@
+            if values:
+                counts[item_id] = self._best_count(values)
+                if len(set(values)) > 1:
+                    logger.warning(
+                        f"材料 {with_template[item_id]['name']}({item_id}) "
+                        f"多次读数不一致 {values}，取 {counts[item_id]}"
+                    )
+            elif item_id in unreadable:
+                logger.warning(f"材料 {with_template[item_id]['name']}({item_id}) 数量识别失败，跳过")
+                skipped.append(item_id)
+            else:
+                logger.warning(f"仓库中未找到材料 {with_template[item_id]['name']}({item_id})，按 0 计")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)